contrib/inventory/gce_googleapiclient.py

```diff
@@ -191,7 +199,14 @@ def main(args):
             if zone:
                 zones_list = [zone_name for zone_name in zone]
             else:
-                zones_list = get_all_zones_in_project(project)
+                for i in range(3):
```

Comment:
create a parameter with a default value for the number of threads. and name `i` as `project_thread` or something better.